backend/src/schema/resolvers/socialMedia.spec.js

```diff
@@ -98,14 +98,19 @@ describe('SocialMedia', () => {
       const variables = {
         url: '',
       }
-      await expect(client.request(mutationC, variables)).rejects.toThrow('Input is not a URL')
+      await expect(client.request(mutationC, variables)).rejects.toThrow(
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-07-03T18:29:14Z" title="Wednesday, July 3rd 2019, 8:29:14 pm +02:00">Jul 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -98,14 +98,19 @@ describe('SocialMedia', () => {
       const variables = {
         url: '',
       }
-      await expect(client.request(mutationC, variables)).rejects.toThrow('Input is not a URL')
+      await expect(client.request(mutationC, variables)).rejects.toThrow(
```


I know you didn't add this, but I have no idea what `mutationC` is in this case
I would need to open up this file to see, which when reviewing a big `PR` makes it even more time-consuming
Not to harp on the same point, but if we get in the habit of instead naming it something generic like `mutation` or `mutationA` then I think we can help ourselves and others to read the code faster.
Or maybe others don't agree with me, in which case I can make an effort to deal with it and not continuously bring it up.

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-07-04T07:58:45Z" title="Thursday, July 4th 2019, 9:58:45 am +02:00">Jul 4, 2019</time>_

---


It is not @roschaefer fault!
`mutationC` is the `CreateSocialMedia` mutation. Today I would name it `mutationCreate` for better readability. The PR would even get bigger for @roschaefer if he would correct al the non optimal naming of me and us.